draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1725,7 +1768,7 @@ primitive, whereas a Mersenne twister pseudorandom number generator {{MT98}} is
 
 - MUST give independent values for distinct (msg, DST, length) inputs.
 Meeting this requirement is subtle.
-As a simplified example, hashing msg || DST does not work,
+As a simplified example, hashing msg \|\| DST does not work,
```

Comment:
Did this not render correctly for you? It was fine in the txt and html files for me.

Reply:
I think this one did, but then I went through and removed all the escapes and some of them didn't render correctly (I think kramdown thought they were starting tables or something weird). So then I went back through and escaped all of them because I wasn't sure which ones would work and which ones wouldn't...